test/test_rate_limit.py

```diff
@@ -0,0 +1,16 @@
+import sendgrid
+import os
+try:
+    import unittest2 as unittest
+except ImportError:
+    import unittest
+
+
+class UnitTests(unittest.TestCase):
+
+    def test_rate_limit(self):
+        sg = sendgrid.SendGridAPIClient(apikey=os.environ.get('SENDGRID_API_KEY'))
+        request_obj = sg.client.access_settings.activity
```

Comment:
@thinkingserious  I checked `python-http-client` repository but couldn't find methods like `set_method` or `get_method`. So how would the user specify which method he wants to use while making the API call? Through parameter?

Reply:
The `python-http-client` uses a [fluent interface](https://sendgrid.com/blog/using-python-to-implement-a-fluent-interface-to-any-rest-api/) with [these methods](https://github.com/sendgrid/python-http-client/blob/master/python_http_client/client.py#L88).